js/board.js

```diff
@@ -1,38 +1,8 @@
 export class Board{
   constructor(){
-    this.dailyDoubleLocations = 0;
-  }
-  populate(inBoardFromDOM) {
-    console.log('Got from DOM', inBoardFromDOM);
-    console.log('Initial', inBoardFromDOM[0].children[0].children[0]);
-    function build() {
-      
-      
-      for (let column = 1; column <= 5;column++){
-      for (let row = 1; row <= 5; row++) {
-        if (row % 5 === 1) {
-          //set the category
-          inBoardFromDOM[0].children[column - 1].children[row - 1].innerText = `Categories${column}`;
-        }
-        if (row % 5 !== 1) {
-          //set clue
-          inBoardFromDOM[0].children[column - 1].children[row - 1].innerText = `Clue${row}`;
-        }
-      }
-    }
-
-    }
-    build();
+    this.dailyDoubleLocations = [];
   }
+  populate(inData) {
```

Comment:
what does `inData` mean? Probably a better parameter name you could use here that's a bit more descriptive...?